SourceCode/Freight/ftt_fr_sales_and_mandate.py

```diff
@@ -0,0 +1,164 @@
+import numpy as np
+import copy
+green_indices=[12, 13, 18, 19]  # Indices for green technologies
+MANDATE_START_YEAR = 2025
+N_YEARS = 11
```

Comment:
In freight N_YEARS should be 16, as the mandate is slower than in the other two sectors.